src/mesh/BoundingBox.cpp

```diff
@@ -108,14 +132,15 @@ int BoundingBox::getDimension() const
 
 const std::vector<double> BoundingBox::dataVector() const
 {
+  std::vector<double> _bounds{_boundMin[0], _boundMax[0], _boundMin[1], _boundMax[1], _boundMin[2], _boundMax[2]};
```

Comment:
This will lead to a segfault, when running in 2D, as `boundMin` is either 2D or 3D, right?

Reply:
Fixed @davidscn